k-distribution/tests/regression-new/set-symbolic-tests/diff-17-spec.k.out

```diff
@@ -0,0 +1,11 @@
+  <k>
+    diffResult ( ( S1
```

Comment:
What should this test be testing?

Reply:
This test does not test any specific simplification. If anything, it tests that simplifications do not over-simplify by matching what they should not match.
As mentioned, I kept all the (comprehensive) tests from the older PR here, and the ones that have correct expected behaviour were kept as tests.